src/models/lucky_cli_version.cr

```diff
@@ -0,0 +1,11 @@
+module LuckyCliVersion
+  extend self
+
+  def current_tag : String
+    "v#{current_version}"
+  end
+
+  def current_version : String
+    "0.15.0"
+  end
```

Comment:
I can see this being useful in the future. For example, I'd like to display the current version on the home page and in the guides somewhere in the future